gpgs/src/utils/LuaUtils.cpp

```diff
@@ -0,0 +1,29 @@
+#include "LuaUtils.h"
+
+bool luaL_checkbool(lua_State *L, int numArg)
+{
+  bool b = false;
```

Comment:
Indentation in this file is different, is this our/your code or someone elses?

Reply:
Yes, this is from my DefVideoAds plugin. I'll fix it, thx!